registration/forms.py

```diff
@@ -238,6 +246,8 @@ def __init__(self, *args, **kwargs):
                 for name, field in self.fields.items():
                     if name not in self.can_be_modified:
                         field.widget.attrs['readonly'] = True
+            else:
+                self.initial['privacy_policy_url'] = settings.DEFAULT_PRIVACY_POLICY_URL
```

Comment:
Why are you setting the default value here?

Reply:
The rationale is that for new instances, we want to require a privacy policy, so I set the initial value here to provide the hint - the user is free to change it if they require. For existing instances that don't already have a privacy policy set (all instances to begin with), we want the registration to be editable without additionally adding a default Privacy Policy that the user hasn't reviewed.